files/Dockerfile

```diff
@@ -28,7 +33,7 @@ RUN apt-get install -y build-essential
 # PHP dependencies
 #
 RUN apt-get update \
- && apt-get -y install gcc git netcat \
+ && apt-get -y install gcc git netcat-openbsd \
```

Comment:
What is `netcat` needed for? It is not needed to build the agent. Maybe this `netcat/netcat-openbsd` dependency could be removed altogether.

Reply:
I don't believe `netcat` is used to build the agent, but it's difficult to rule out that it isn't used in execution in some way or another (searching for `nc` against the repo returns a couple thousand results). It could have been used historically for testing agent functionality and may not be necessary anymore, but I don't think it hurts anything to keep it included for the time being.

Reply:
Agree. I did check that `netcat` package in Debian 11 is a virtual package that redirects to `netcat-openbsd` so this is not a breaking change and I'm ok with leaving it as is.